docs/content/preview/api/ysql/exprs/geo_partitioning_helper_functions/func_yb_is_local_table.md

```diff
@@ -36,20 +36,20 @@ This function is helpful while implementing [Row-level geo-partitioning](../../.
     ```sh
     ./bin/yugabyted start                           \
       --base_dir=/home/yugabyte/<IP1>/yugabyte-data \
-      --listen=<IP1>                                \
+      --advertise_address=<IP1>                                \
       --master_flags "placement_cloud=aws,placement_region=us-west-1,placement_zone=us-west-1c" \
```

Comment:
@nmalladi These master/tserver placement flags - are they now replaced by yugabyted --cloud_location flag
@sudhanshu456 see https://docs.yugabyte.com/preview/reference/configuration/yugabyted/#create-a-multi-region-cluster

Reply:
Was there any mention of a replacement for --cloud_location flag in our docs, or we just replaced it?

Reply:
The example is old, this would have been several versions back - pre 2.14 I'm thinking

Reply:
I don't think docs page is being used anymore right?
As these docs are target for preview release, we should use the'-cloud_location` flag to pass the placement info.